demos/github-demo/src/isograph-components/UserLink.tsx

```diff
@@ -4,11 +4,10 @@ import type { ResolverParameterType as UserLinkParams } from '@iso/Actor/UserLin
 
 import { Link } from '@mui/material';
 
-export const UserLink = iso<UserLinkParams>`
-  field Actor.UserLink @component {
+export const UserLink = iso(` field Actor.UserLink @component {
     login,
   }
-`(UserLinkComponent);
+`)(UserLinkComponent);
 
 function UserLinkComponent(props: UserLinkParams) {
```

Comment:
it looks like if the function is defined separately, it's arg type is not inferred, so we need to specifically say `UserLinkParams` here